tests/data/config/_init.yaml

```diff
@@ -9,3 +9,10 @@ avg_filter_active: false
 tcp_port: 9999
 host: 127.0.0.1
 timeout: 2
+model_id: "9517b"
+location: "Simulation"
+sensor_brand: "?"
```

Comment:
best to make these unique.
sensor_brand: "test_sensor_mitutoyo"
sensor_model: "test_sensor_S2251"
sensor_serial: "asdf1234"